autotest/ogr/ogr_odbc.py

```diff
@@ -173,15 +173,58 @@ def test_extensions():
         ds = ogrtest.odbc_drv.Open('data/mdb/empty.accdb')
         assert ds is not None
 
+
 ###############################################################################
-# Cleanup
+# Test reading mdb with null memo fields (https://github.com/OSGeo/gdal/pull/3458)
 
 
-def test_ogr_odbc_cleanup():
+def test_null_memo():
     if ogrtest.odbc_drv is None:
         pytest.skip()
 
-    gdal.Unlink('tmp/odbc.mdb')
+    ds = ogrtest.odbc_drv.Open('data/mdb/bad.mdb')
```

Comment:
can the file be renamed to something like null_memo.mdb to be more explicit about its content ?

Reply:
done!